compiler/src/dmd/cxxfrontend.d

```diff
@@ -256,9 +256,9 @@ Expression getDefaultValue(EnumDeclaration ed, Loc loc)
 /***********************************************************
  * expression.d
  */
-void expandTuples(Expressions* exps, Identifiers* names = null)
+void expandTuples(Expressions* exps, ArgumentLabels* argLabels = null)
 {
-    return dmd.expression.expandTuples(exps, names);
+    return dmd.expression.expandTuples(exps, argLabels);
 }
```

Comment:
I don't understand how is this passing the testsuite. Are new symbols being added without link tests?